scripts/deploy.py

```diff
@@ -110,54 +124,50 @@ def UpdateChangeLog(p):
     p.AddStep(RunStoreFileSha(CHANGELOG_PATH))
     p.AddStep(
         RunCmdStep(
-            '%s %s' % (os.getenv('EDITOR', 'vim'), CHANGELOG_PATH),
-            doc="Edit changelog"))
+            '{0} {1}'.format(os.getenv('EDITOR', 'vim'), CHANGELOG_PATH),
+            doc='Edit changelog'))
     p.AddStep(CompareFileSha(CHANGELOG_PATH, require_equal=False))
-    p.AddStep(RunCmdStep('git add %s' % CHANGELOG_PATH))
+    p.AddStep(RunCmdStep(f'git add {CHANGELOG_PATH}'))
```

Comment:
If we're going to use `f` strings, then we need to be sure all release devs have 3.6 available, and if so, then all possible formatting should be done with this (e.g. two lines above). Otherwise, use `format()` exclusively.

Reply:
I'm pretty sure both I and Error have python 3.7 available.
I use f'' where it's more readable and format (or I'd actually prefer `%`) where it's not.